python/cugraph/cugraph/sampling/random_walks.py

```diff
@@ -96,6 +120,7 @@ def random_walks(
     >>> _, _, _ = cugraph.random_walks(G, "uniform", start_vertices, 3)
 
     """
+
     if legacy_result_type:
         warning_msg = (
             "Coalesced path results is deprecated and will no longer be "
```

Comment:
I know this wasn't in this PR, but I think the deprecation message should include the specific param that is being deprecated, so in this case it should mention "legacy_result_type", maybe like this:
```suggestion
            "Coalesced path results, returned when setting legacy_result_type=True, is deprecated and will no longer be "
```